packages/cache/src/internal/cacheHttpClient.ts

```diff
@@ -31,12 +31,7 @@ import {
 const versionSalt = '1.0'
 
 function getCacheApiUrl(resource: string): string {
-  // Ideally we just use ACTIONS_CACHE_URL
-  const baseUrl: string = (
-    process.env['ACTIONS_CACHE_URL'] ||
-    process.env['ACTIONS_RUNTIME_URL'] ||
```

Comment:
Breaking change here.  If we're going to remove this, we need to bump the package version to `2.0.0`

Reply:
This should not be a breaking change as we are already setting ACTIONS_CACHE_URL in runner based on the presence of Artifact Cache service https://github.com/github/actions-dotnet/blob/9925858070aba79b8c5f8be793f58e1117ecd03d/Actions/Runtime/Sdk/Server/TaskHub.cs#L3484  and https://github.com/actions/runner/blob/main/src/Runner.Worker/Handlers/NodeScriptActionHandler.cs#L53-L56

Reply:
@aiqiaoy  for 👀

Reply:
While technically yes it is a breaking change, pretty sure it is not so in practical terms. In hosted Actions, `ACTIONS_CACHE_URL` is set and no-one would be using the fallback. In GHES, caching was not available and hence need not worry about older server/runner versions.
Hence we should be okay to do without extra work of a major version change.

Reply:
updated for major release